backend/src/nginx/diagnostic-ui.conf

```diff
@@ -1,3 +1,9 @@
+
+map $http_upgrade $connection_upgrade {
```

Comment:
:disappointed:  I don't know enough about this conf to say if it is good or not. Would like a proof that it did work.